capa/features/extractors/base_extractor.py

```diff
@@ -53,14 +89,14 @@ def __init__(self):
         super(FeatureExtractor, self).__init__()
 
     @abc.abstractmethod
-    def get_base_address(self) -> int:
```

Comment:
`None` for .NET or shellcode?

Reply:
- [ ] TODO: decide on this

Reply:
af9049da6e106c1da4dbcb6fa0a972f126bfb1fa